### PR TITLE
Add 'require "mobility/ransack"' into plugin file

### PR DESCRIPTION
**Pull request by adrianryt**
This PR is related to issue https://github.com/shioyama/mobility-ransack/issues/25

This might be connected to Rails 7 autoloading changes

### CODE REVIEW

**Comment by adrianryt** [comment]
@shioyama Any thoughts?

**Comment by shioyama** [comment]
Hmm, strange. I'm not sure why that would have changed, but this is fine. Thanks for making the PR! I'm running CI now on this.

File: lib/mobility/plugins/ransack.rb [hunk 1]
```diff
@@ -21,7 +22,6 @@ module Ransack
         end
       end
     end
-
```
Comment (shioyama) [inline lib/mobility/plugins/ransack.rb:24]: Hmm this blank line was intentional.

**Comment by shioyama** [comment]
Merged in b49710323c64198192687e8f3f8ec8ca0d54e8a5

**Pull request closed without merging**